### ISSUE TITLE
Geometry not visible in Dynamo Studio 1.3.0.875

### ISSUE THREAD

**Issue by maciejwypych** [DynamoDS/Dynamo#7964]
## Dynamo version

Dynamo Studio 1.3.0.875

## Operating system

Windows 7 64bit

## What did you do? 

Opened Dynamo Studio and created geometry

## What did you expect to see?

Geometry

## What did you see instead?

Nothing

I can't see any geometry I create in Dynamo Studio. Dynamo for Revit works fine, but in Dynamo Studio it doesn't appear.
I've created a point by coordinates and I can see the Axis when I select it, but I can't see the point, or any other geometry for that matter.

**Comment by jnealb** [comment]
@maciejwypych Do you see the 3D grid or truly nothing at all?

**Comment by maciejwypych** [comment]
I can see the grids and XYZ axis if I place a point, but no geometry.(and
the geometry is definitely created)

**Comment by jnealb** [comment]
Sounds like another case of DYN-887 - "Dynamo install/upgrade process leads to disabled gfx rendering".
Can you please try un-installing Studio completely and then re-installing it?

**Comment by maciejwypych** [comment]
Hi Neal,

I've reinstalled Dynamo Studio, but the problem is still there.

No geometry visible.

As a comparison that's what I can see in Dynamo Studio


And the same definition in Dynamo Revit


Thanks,

Maciej
​
​

**Comment by maciejwypych** [comment]
Looks like uninstalling and reinstalling both Dynamo Revit and Dynamo Studio, has done the trick.

**Comment by maciejwypych** [comment]
Once, I've opened Dynamo for Revit, restarted and opened Dynamo Studio it doesn't display the geometry anymore.

**Comment by jnealb** [comment]
Does Studio not support geometry if you do not first launch Dynamo for Revit? In other words, reboot and start Studio: does it work?

**Comment by maciejwypych** [comment]
That's what I've done.
It worked just after I've reinstalled both Dynamo for Revit and Dynamo
Studio.
But after I've restarted and launched Dynamo for Revit, closed Revit and
then opened Dynamo Studio, geometry wasn't visible anymore.
I've restarted and tried launching Dynamo Studio first, but the geometry is
not visible anymore.

I've repeated the process, and if I open Dynamo for Revit at any point,
Dynamo Studio stops showing geometry.
